src/fmu/dataio/scripts/create_case_metadata.py

```diff
@@ -20,7 +20,6 @@
 
 from fmu.config._loader import FmuLoader
```

Comment:
There is still a dependency on fmu.config here. It is used for the FmuLoader class, but I think actually this is not needed here; just use normal YAML safeload as it it is the **output** FMU config which is parsed here (which shall be standards YAML)

Reply:
Using native yaml loader